src/app/Services/Videos/VideoService.php

```diff
@@ -17,4 +18,22 @@ public function generateVkey(): string
     {
         return Str::random(14);
     }
+
+    public function extractMeta(string $path): array {
+        $ffmpeg = FFMpeg::create();
+        $video = $ffmpeg->open($path);
```

Comment:
This can possible throw errors in case of bad file paths. Wrap the whole method in try/catch.